src/test/scala/io/iohk/ethereum/mpt/MerklePatriciaTrieSuite.scala

```diff
@@ -572,25 +573,15 @@ class MerklePatriciaTrieSuite extends AnyFunSuite with ScalaCheckPropertyChecks
         }
 
       // when
-      val proof: Option[Vector[MptNode]] = trie.getProof(keyToFind)
+      val proofOpt: Option[Vector[MptNode]] = trie.getProof(keyToFind)
 
       // then we can get proof if we know key exist
-      assert(proof.isDefined)
-
+      assert(proofOpt.isDefined)
       // then we can recreate MPT and get value using this key
-      val nodeStorage: NodeStorage = proof.get.foldLeft(emptyNodeStorage) { case (storage, node) =>
-        val k = ByteString(node.hash)
-        val v = node.encode
-        storage.put(k, v)
+      proofOpt.map{ p =>
+        val ver = verifyProof[Array[Byte],Array[Byte]](trie.getRootHash, keyToFind, p)
```

Comment:
a comma would be nice in here
```suggestion
        val ver = verifyProof[Array[Byte], Array[Byte]](trie.getRootHash, keyToFind, p)
```